spec/views/cases/show_html_slim_spec.rb

```diff
@@ -310,4 +313,62 @@ def setup_policies(policies)
       end
     end
   end
+
+  describe 'SAR case' do
```

Comment:
what is this test actually testing for?

Reply:
I read it in conjunction with the context so that the first context would read `SAR case before it is extended`. Is there a style guide for writing specs/preference?

Reply:
No style guide, just should be understandable, we check for that in peer review. I was just saying I fing it a little unclear I feel the describe should be 'extend SAR deadline link' or something, that seems like the thing we are testing here

Reply:
Renamed spec name to be more descriptive